website/content/docs/v0.12/Bare Metal Platforms/digital-rebar.md

```diff
@@ -0,0 +1,172 @@
+controlplane.yaml is valid for metal mode
+$ talosctl validate --config join.yaml --mode metal
+join.yaml is valid for metal mode
+```
+
+#### Publishing the Machine Configuration Files
+
+Digital Rebar has a build-in fileserver, which means we can use this feature to expose the talos configuration files.
+We will place `controlplane.yaml`, and `worker.yaml` into Digital Rebar file server by using the `drpcli` tools.
+
+Copy the generated files from the step above into your Digital Rebar installation.
+
+```bash
+drpcli file upload <file>.yaml as <file>.yaml
+```
+
+Replacing `<file>` with controlplane or worker.
+
+### Download the boot files
+
+Download a recent version of `boot.tar.gz` from [github.](https://github.com/talos-systems/talos/releases/)
+
+Upload to DRB:
+
+```bash
+$ drpcli isos upload boot.tar.gz as talos.tar.gz
+{
+  "Path": "talos.tar.gz",
+  "Size": 96470072
+}
+```
+
+We have some Digital Rebar [example files](https://github.com/talos-systems/talos/tree/master/hack/test/digitalrebar/) in the Git repo you can use to provision Digital Rebar with drpcli.
+
+To apply these configs you need to create them, and then apply them as follow:
+
+```bash
+$ drpcli bootenvs create talos
+{
+  "Available": true,
+  "BootParams": "",
+  "Bundle": "",
+  "Description": "",
+  "Documentation": "",
+  "Endpoint": "",
+  "Errors": [],
+  "Initrds": [],
+  "Kernel": "",
+  "Meta": {},
+  "Name": "talos",
+  "OS": {
+    "Codename": "",
+    "Family": "",
+    "IsoFile": "",
+    "IsoSha256": "",
+    "IsoUrl": "",
+    "Name": "",
+    "SupportedArchitectures": {},
+    "Version": ""
+  },
+  "OnlyUnknown": false,
+  "OptionalParams": [],
+  "ReadOnly": false,
+  "RequiredParams": [],
+  "Templates": [],
+  "Validated": true
+}
+```
+
+```bash
+drpcli bootenvs update talos - < bootenv.yaml
+```
+
+> You need to do this for all files in the example directory.
+> If you don't have access to the `drpcli` tools you can also use the webinterface.
+
+It's important to have a corresponding SHA256 hash matching the boot.tar.gz
+
+#### Bootenv BootParams
+
+We're using some of Digital Rebar build in templating to make sure the machine gets the correct role assigned.
+
+`talos.platform=metal talos.config={{ .ProvisionerURL }}/files/{{.Param \"talos/role\"}}.yaml"`
+
+This is why we also include a `params.yaml` in the example directory to make sure the role is set to one of the following:
+
+- controlplane
+- worker
+
+The `{{.Param \"talos/role\"}}` then gets populated with one of the above roles.
+
+### Boot the Machines
+
+In the UI of Digital Rebar you need to select the machines you want te provision.
+Once selected, you need to assign to following:
+
+- Profile
+- Workflow
+
+This will provision the Stage and Bootenv with the talos values.
+Once this is done, you can boot the machine.
+
+To understand the boot process, we have a higher level overview located at [metal overview.](/v0.11/en/guides/metal/overview)
```

Comment:
we should update `/v0.11/` to `/v0.12` here

Reply:
Converted to a relative URL instead